Src/Base/AMReX_Arena.H

```diff
@@ -102,6 +102,11 @@ public:
 
     static const std::size_t align_size = 16;
 
+    /**
+     *  \brief Return the ArenaInfo object for querying
+     */
+    const ArenaInfo& the_arena_info() const { return arena_info; }
```

Comment:
Can we rename this to `arenaInfo` to be consistent with how we name other methods (e.g., BaseFab::arena, MultiFab::boxArray)?